java/client/src/org/openqa/selenium/internal/Require.java

```diff
@@ -159,6 +159,20 @@ public static int positive(String argName, Integer number, String message) {
     return number;
   }
 
+  public static double positive(String argName, Double number, String message) {
```

Comment:
Since you're returning a `double`, you should make `number` a primitive type and not a wrapper type. This prevents an unnecessary boxing from occurring and obviates the need for the null check.